apps/remix-dapp/src/locales/en/udapp.json

```diff
@@ -61,13 +61,13 @@
   "udapp.reset": "Reset",
   "udapp.delete": "Delete",
   "udapp.injectedTitle": "Unfortunately it's not possible to create an account using injected provider. Please create the account directly from your provider (i.e metamask or other of the same type).",
-  "udapp.createSmartAccount": "Create Safe Smart Account",
-  "udapp.createSmartAccountAlpha": "Create Safe Smart Account [ALPHA]",
+  "udapp.createSmartAccount": "Create Gnosis Safe Smart Account",
+  "udapp.createSmartAccountAlpha": "Create Gnosis Safe Smart Account [ALPHA]",
   "udapp.createSmartAccountDesc1":"ERC-4337 introduces smart contracts accounts, which are more advanced than traditional EOAs. These accounts offer features like MULTISIGNATURE for extra security, GASLESS transactions and custom transaction rules for added flexibilitity.",
```

Comment:
ERC-4337 introduces smart contracts accounts, which are more advanced than traditional EOAs. These accounts offer features like multi-signature functionality, gasless transactions and custom transaction rules.